README.md

```diff
@@ -16,7 +16,12 @@ How to build locally
 
 We recommend to install the dependencies in a virtual environment.
 
-* Create an environment e.g. a [Conda](https://docs.conda.io/en/latest/) and activate it.
+* Create an environment e.g. using [Conda](https://docs.conda.io/en/latest/), activate it and install pip:
+
+  ``$ conda create -n omeroguides``
```

Comment:
it is preferable to specify a python 3 version otherwise an env with python 2.7 is created.
Also specifying for example python=3.6, ``pip `` is automatically installed.

Reply:
Done in a076be7